baker/SiteBaker.tsx

```diff
@@ -832,6 +831,10 @@ export class SiteBaker {
             }
             try {
                 await this.bakeOwidGdoc(dataInsight)
+                // We don't need latest data insights nor their images on the
+                // index page.
+                dataInsight.latestDataInsights = []
+                dataInsight.imageMetadata = attachments.imageMetadata
```

Comment:
This confused me just a little bit because I was expecting this adjustment to be done right next to the `renderDataInsightsIndexPage` call.

Would you mind clarifying that? 🙂

Reply:
I tweaked it a bit, let me know if it's good enough. I kept it in the loop, because that's where we have the `attachments.imageMetadata` for each data insight.